service/settings.toml

```
@@ -5,3 +5,4 @@ DATABASE_URL = "sqlite:///:memory:"
 SEARCH_QUERY= 'topic:"flask-extension flask-extensions"'
```

Parece que é nesta linha: https://github.com/BurntSushi/toml-test/blob/master/tests/valid/string-escapes.toml

```suggestion
SEARCH_QUERY= "topic:'flask-extension flask-extensions'"
```

---

Sugestão do @rochacbruno